### ISSUE TITLE
Pass to adoptium

### ISSUE THREAD

**Issue by LvffY** [jyksnw/install-jdk#15]
It seems that the current URL **api.adoptopenjdk.net** is deprecated (see [home page](https://api.adoptopenjdk.net/)), and that the URL should be changed to **api.adoptium.net**. 

There is a [migration guide](https://api.adoptopenjdk.net/migration.html) that could be follow.

**Comment by LvffY** [comment]
If I'm right, the line to change should be https://github.com/jyksnw/install-jdk/blob/b45f5bf8260812e2bcc8727c5ff297cad952a6c6/jdk/__init__.py#L56

**Comment by LvffY** [comment]
@jyksnw did you have time to check on this ?

**Comment by jyksnw** [comment]
Hi @LvffY, I am short on time these days to maintain this repository. Feel free to fork it or put in a PR to resolve this issue. Otherwise, I will look at the changes within the next month to determine what needs to be updated.

**Comment by jyksnw** [comment]
Just an update on this - I will be dedicating time over the next weeks and months to bring new life to this project, where the priority will be updated to use the Adoptium.

**Comment by jyksnw** [comment]
@LvffY I have updated the main branch to switch away from AdoptOpenJDK and have elected to use a pluggable client system that allows the end user to select which build implementation to choose from. Adoptium is now the default, and I have the option to switch to using Corretto. I did make some substantial code changes behind the scenes but left the primary client API intact to be backward compatible with prior releases. Because of this, I will be testing this out before pushing it up to PyPI.

I would love for others to checkout the main branch and do some additional real-world testing.
